distributions/aws/test/e2e/utils/cloudformation_resources.py

```diff
@@ -0,0 +1,131 @@
+from e2e.utils.config import configure_resource_fixture
+
+
+def create_stack(
+    cfn_client,
+    template_path,
+    stack_name,
+    params={},
+    capabilities=[],
+    timeout=300,
+    interval=10,
```

Comment:
nit: interval to 30? same for other places

Reply:
sure